### ISSUE TITLE
Price for sTLOS

### ISSUE THREAD

**Issue by pmjanus** [telosnetwork/telos-wallet#330]
Can we calculate the price of sTLOS based on the conversion ratio and price of TLOS.

## Acceptance Criteria:
- Calculate price for sTLOS using conversion ratio and price of wTLOS and use it for all fiat price displays of sTLOS

**Comment by Viterbo** [comment]
Two things about this issue:
![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/7a379d12-1275-4337-8fa8-cbb0083a3418)

1 - we don't need this since we already have the indexer bringing not only that info for the token but a lot of interesting stuff related to the markets of that token (See picture)

2 - My PR includes this new indexer-based way of fetching the balances (which is using a super-fast single query) but maintaining the old functionality as a fallback for blockchains that don't have indexer support (yet). This solution is hardcoded but we can make it dynamic, so if we detect that the indexer is down, we can switch temporarily to the fallback solution where this issue has a place.

In summary, it is a good idea to include as a fallback for blockchains not having support for indexers. But EVM-Mainnet does, so that fact puts this issue in low priority.

Does it make sense?
